src/org/labkey/test/tests/issues/IssuesAdminTest.java

```diff
@@ -154,19 +167,102 @@ public void customIssueNameTest()
         assertElementNotPresent(Locator.lkButton("New " + defaultSingular));
     }
 
-    // @Test @Ignore //TODO
+    @Test
     public void testProtectedFields() throws Exception
     {
+        goToProjectHome();
+        clickAndWait(Locator.linkWithText(ISSUE_LIST_NAME));
+        IssuesAdminPage issuesAdminPage = _issuesHelper.goToAdmin();
+
+        log("Checking for the protected field");
+        DomainFormPanel domainFormPanel = issuesAdminPage.getFieldsPanel();
+        checker().verifyTrue("Title should be disabled", domainFormPanel.getField("Title").isFieldProtected());
+        checker().verifyTrue("NotifyList should be disabled", domainFormPanel.getField("NotifyList").isFieldProtected());
+        checker().verifyTrue("AssignedTo should be disabled", domainFormPanel.getField("AssignedTo").isFieldProtected());
+        checker().verifyTrue("Resolution should be disabled", domainFormPanel.getField("Resolution").isFieldProtected());
```

Comment:
may be worth adding a couple of checks that some of the other default fields are not protected